Apps/W1/EDocument/app/src/InvoicePreview/EInvoiceLine.Table.al

```diff
@@ -0,0 +1,53 @@
+table 6100 "E-Invoice Line"
+{
+    DataClassification = CustomerContent;
+
+    fields
```

Comment:
Shouldn't we add ToolTips for new tables as default on the Table instead of the page?